quiz/json_response_mixin.py

```diff
@@ -0,0 +1,24 @@
+'''
+This module contains a JSON Response Mixin.
+'''
+
+from django.http import JsonResponse
+
+class JSONResponseMixin(object):
```

Comment:
I'm going to put this one on hold.  As I'm ditching the class based view until I get something working.  I'll swing back if there's still time.